### ISSUE TITLE
Faile to display some math symbols in debug console and comment out code line/block.

### ISSUE THREAD

**Issue by hongyi-zhao** [kenkangxgwe/lsp-wl#50]
On Ubuntu 20.04, with the latest stable vscode and git master version of lsp-wl, see the following screenshot for more detailed info: 

![image](https://user-images.githubusercontent.com/11155854/119770959-83a4aa00-beef-11eb-8446-b7057b14d1b8.png)

Another problem, I find the "Toggle Code Line/Block Comment" don't work at all.
 
Any hints for these problems?

Regards,
HY

**Comment by kenkangxgwe** [comment]
Hi Hongyi,
I think it failed because `\[HumpEqual]` is not a valid expression. You will get the same result if you run it in MMA. I think what you want to run is `"\[HumpEqual]"`, right?

**Comment by kenkangxgwe** [comment]
"Toggle Code Line/Block Comment" is provided by the syntax highlight plugin. LSP is not in charge of that feature, you may want to install https://marketplace.visualstudio.com/items?itemName=shigma.vscode-wl or https://marketplace.visualstudio.com/items?itemName=flipphillips.wolfram-language

**Comment by hongyi-zhao** [comment]
> Hi Hongyi,
> I think it failed because `\[HumpEqual]` is not a valid expression. You will get the same result if you run it in MMA. I think what you want to run is `"\[HumpEqual]"`, right?

You're partially right. To be more specific, `\[HumpEqual]` is a valid expression in MMA based on my testing with the following script:
```
werner@X10DAi:~$ cat 11.wl 
(* ::Package:: *)

(* ::Title:: *)
(*Title of the file*)

(* ::Section:: *)
(*Section 1*)

2+5 

"\[HumpEqual]"
"\[Eta]"

\[HumpEqual]
```
See below for more detailed info:

![image](https://user-images.githubusercontent.com/11155854/119839641-e53c3700-bf36-11eb-91da-6a3f8dcce290.png)

But only the double-quoted style will be displayed in `vscode` correctly when using it alone, as shown below:

![image](https://user-images.githubusercontent.com/11155854/119825563-a94ea500-bf29-11eb-91aa-83fd70afbe2f.png)

To sum up, `\[HumpEqual]` in nature is a binary operator, so it must be used like this: `a\[HumpEqual]b`, otherwise, it will be treated as the corresponding character only, so double-quoted version will work in `vscode`. See my following testing in `vscode`:

![image](https://user-images.githubusercontent.com/11155854/119842077-f5edac80-bf38-11eb-8dd4-7a1a497f7805.png)

**Comment by hongyi-zhao** [comment]
> "Toggle Code Line/Block Comment" is provided by the syntax highlight plugin. LSP is not in charge of that feature, you may want to install https://marketplace.visualstudio.com/items?itemName=shigma.vscode-wl or https://marketplace.visualstudio.com/items?itemName=flipphillips.wolfram-language

Really, the above two extensions are the most popular ones among this type as represented [here](https://marketplace.visualstudio.com/search?term=wolfram&target=VSCode&category=All%20categories&sortBy=Relevance). I installed both of them, and they work like a charm.
